### ISSUE TITLE
IPC brains still not working correctly

### ISSUE THREAD

**Issue by Bloofershmoo** [yogstation13/Yogstation#20628]
### Severity

Round Breaking

### Short Description

IPC brains do not work correctly, and do not support transplants

### Reproduction

1. Find IPC
2. Remove brain Via surgery or violence
3. examine brain
4. brain should say "this one seems particularly lifeless, perhaps it will regain its luster later" or something, i don't remember exactly
5. place IPC brain in another IPC, or other medium that supports posibrain
6. Whatever it's placed in will not gain consciousness 

### Round ID

Round #49742

### Test Merges

#20501 #20547 #20569

### BYOND Username

Bloofershmoo

### Was this done on a locally hosted or non-Yogstation hosted server?

No

### Additional information

I'm fairly confident the IPC's soul is in the chest that houses the posibrain, rather than the posibrain itself, as during previous encounters with this bug, as the IPC, while spectating my own corpse, returning to body would always send me back to my torso, even after my head was removed, and once gibbed, returning to body would give the message "You have no body." While I do know this bug has been reported a couple times now, I just thought I would try to bring attention back to it when I encountered it again recently.

**Comment by MajManatee** [comment]
4.s text means they like closed the game, an IPCs head has nothing important to them, since yeah the brain is stored in the chest. If you were gibbed then yeah you wouldn't have a body. I haven't heard of this issue before myself.

**Comment by Bloofershmoo** [comment]
I specified the text seen when examining the brain as when examined, the brain meets all of the conditions to have a soul, as i was the ghost of the IPC examining my own brain and seeing this text, and examining the body before brain removal would have no issues. I felt the need to clarify that IPC soul is not in the head, as a previous bug report on this issue, report #20336. (I believe this is the first report on this issue) claims that the IPC soul transfers to the head when gibbed, but i don't believe that is correct. I figured the return to body function saying "You have no body" would be another product of this bug, as with other races, the return to body function returns you to your brain if it is not in your body, but with IPCs the function will never return you to your brain, and once the torso is destroyed, the IPC seems to be round removed, and impossible to revive. Though i don't think it provides much more info or help on this bug, report #20416 is another report on this issue.
